src/Resources/Locales/en.json

```diff
@@ -622,6 +622,42 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "powerBI": {
+            "title": "Power BI widget",
+            "libraryName": "Power BI Visual",
+            "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor",
+            "type": {
+                "label": "Type",
+                "options": {
+                    "report": "Report",
+                    "dashboard": "Dashboard",
+                    "tile": "Tile",
+                    "visual": "Visual"
+                }
+            },
+            "reportUrl": {
+                "label": "Report URL",
+                "placeholder": "https://embed.powerbi.com/groups/me/reports/{reportId}",
+                "description": "The URL used to embed report artifacts in external tools"
+            },
+            "pageName": {
+                "label": "Page Name",
+                "placeholder": "Page name"
+            },
+            "visualName": {
+                "label": "Visual Name",
+                "placeholder": "Visual name",
+                "suffix": "Enter the name of the specific element from the report to show in this widget"
+            },
+            "dataFilters": {
+                "label": "Data Filters"
+            },
+            "errors": {
+                "missingConfiguration": "Widget configruation is missing or is invalid",
+                "missingAdapter": "Please ensure that the Power BI adapter is installed and configured",
```

Comment:
ideally the user would not see this since it won't mean much to them. 🤔

Reply:
Correct. This is usually do to consumer of cardboard package having something misconfigured.